src/MathProg/optimizerwrappers.jl

```diff
@@ -118,6 +118,8 @@ function sync_solver!(optimizer::MoiOptimizer, f::Formulation)
     for ((c_id, v_id), coeff) in buffer.reset_coeffs
         # Ignore modifications involving vc's that were removed
         (c_id in buffer.constr_buffer.removed || v_id in buffer.var_buffer.removed) && continue
+        iscuractive(f, c_id)&&iscuractive(f, v_id) || continue
+        isexplicit(f, c_id)&&isexplicit(f, v_id) || continue
```

Comment:
```suggestion
        iscuractive(f, c_id) && isexplicit(f, c_id) || continue
        iscuractive(f, v_id) && isexplicit(f, v_id) || continue
```

We always check if a constraint or a variable is active and explicit so it's better if we keep this informal convention here